### PR TITLE
Sandbox

### PR DESCRIPTION
**Pull request by PhilVest**
Added stress calculation options. More additions for calculating the principal stresses will be given in the future.

### CODE REVIEW

File: s3dxrd/utils/stiffness.py [hunk 1]
```diff
@@ -0,0 +1,134 @@
+import numpy as np
+
+
+def stiff_mat():
```
Comment (AxelHenningsson) [inline s3dxrd/utils/stiffness.py]: can we name this something with alpha quartz?

File: s3dxrd/utils/stiffness.py [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+    C[3, 3] = C44
+    C[4, 4:6] = [C44, C14]
+    C[5, 5] = C66
+
+    C = C + np.tril(C.T, -1)
+    C = C*(10**9) # Convert to GPa
+    return C
+
+def rotate_stiffness(U, C):
+    """Rotate a series of 6x6 "tensor matrices" according to a series of 3x3 rotation matrices.
```
Comment (AxelHenningsson) [inline s3dxrd/utils/stiffness.py]: Can we have these type of docstrings on all functions?

File: s3dxrd/utils/stiffness.py [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+    M = _get_rotation_matrix(U)
+    C_rot = (M @ C @ np.transpose(M))
+
+    return C_rot
+
+
+def _get_rotation_matrix(U):
+    """Return the rotation matrix, R, that corresponds to rotating the stiffness tensor by the rotation defined by U
+
+    R*eps_bar contains the same values in vector format where eps_bar is 6x1 stack of eps uniqie values as
```
Comment (AxelHenningsson) [inline s3dxrd/utils/stiffness.py]: What format?

**Review by AxelHenningsson — COMMENTED** [review]
Nice! Some minor things commented.

**Comment by AxelHenningsson** [comment]
(I am also thinking that we should perhaps have some tests (unittests) in this package to make life easier. But perhaps we are not there yet.)

**Comment by PhilVest** [comment]
Added some docstrings to the code, improved the names of functions and removed redundant comments.

**Comment by AxelHenningsson** [comment]
Excellent

**Pull request merged**